base/xz/stone.yml

```diff
@@ -4,18 +4,18 @@
 # SPDX-License-Identifier: Zlib
 #
 name        : xz
-version     : 5.2.7
-release     : 2
```

Comment:
So for now we're not bumping the relnum for an update to a package that only exists in venom. The guide is that the relnum should be `1` if the package only exists in venom, and `2` if it exists in main. 

The reason for this is that we are going to "play" out the packages/updates in this repo on top of main once main is open for package submissions. So it doesn't matter how many times the package in venom is updated, we're going to update the "main" package just once with whatever is the latest in venom.

Reply:
Okay, got it